### PR TITLE
Jump to edited block

### PR DESCRIPTION
**Pull request by JulianKniephoff**
Like we discussed with Lisa today.

Note that this includes https://github.com/elan-ev/tobira/pull/377 for no particular reason other than my own convenience. :stuck_out_tongue_winking_eye:

### CODE REVIEW

File: frontend/src/routes/manage/Realm/Content/index.tsx [hunk 1]
```diff
@@ -115,9 +116,21 @@ const ManageContent: React.FC<Props> = ({ data }) => {
...
+    const hasUnsavedChanges = editedBlock !== undefined;
     useNavBlocker(hasUnsavedChanges);
 
+
+    // When a block goes into edit mode, we want to scroll it into view
+    const blockRefs = useRef(new Map<string, HTMLDivElement>());
+    useEffect(() => {
+        if (hasUnsavedChanges) {
+            const ref = blockRefs.current.get(editedBlock.id) ?? bug("unbound ref");
+            ref.scrollIntoView({ behavior: "smooth" });
```
Comment (LukasKalbertodt) [inline frontend/src/routes/manage/Realm/Content/index.tsx:129]:
```suggestion
            ref.scrollIntoView({ behavior: "smooth", block: "center" });
```

I would scroll it into the middle of the view. I find it kind of weird to have the div, especially if it's not very high, to be at the very top. It gets more confusing the larger your screen is.

**Review by LukasKalbertodt — CHANGES_REQUESTED** [review]
Just a UX suggestion.

File: frontend/src/routes/manage/Realm/Content/index.tsx (hunk 1 above)
Comment (JulianKniephoff) [reply frontend/src/routes/manage/Realm/Content/index.tsx:129]: Oh cool, I didn't know about that. :D

**Review by LukasKalbertodt — APPROVED** [review]
Noice

**Pull request merged**